pages/dao/[symbol]/proposal/components/instructions/raydium/AddLiquidity.tsx

```diff
@@ -49,26 +49,19 @@ const AddLiquidityRaydium = ({
     governedTokenAccounts,
     getMintWithGovernances,
   } = useGovernanceAssets()
-  // const governedProgramAccounts = getGovernancesByAccountType(
-  //   GovernanceAccountType.ProgramGovernance
-  // ).map((x) => {
-  //   return {
-  //     governance: x,
-  //   }
-  // })
 
   useEffect(() => {
     async function prepGovernances() {
       const mintWithGovernances = await getMintWithGovernances()
       const matchedGovernances = governancesArray.map((gov) => {
-        const governedTokenAccount = governedTokenAccounts.find(
+        const governedbaseccount = governedTokenAccounts.find(
```

Comment:
`governedbaseccount` is `governedBaseAccount`?

Reply:
yes, nice catch!